CHANGELOG

```diff
@@ -1,3 +1,7 @@
+snowplow-media-player 0.3.2 (2022-10-26)
+---------------------------------------
+Point docs to core docs site (docs.snowplow.io)
```

Comment:
Every change needs to be related to an issue, so looks like you will first need to create an issue for 'Point docs to core docs site', then we can mark it as (Close #...)